### ISSUE TITLE
Movement calculation, etc.

### ISSUE THREAD

**Issue by AdmiralNyar** [Miskatonic-Investigative-Society/CoC7-FoundryVTT#53]
Hello HavlockV. I'm trying to translate your wonderful game system into Japanese.
In that process, I found an error in the automatic calculation of MOV.

As stated on page 33 of the rulebook, if STR=DEX=SIZ, the Mov is 8.
So I'd like to ask you to fix the following code, is that ok?

Code:
https://github.com/HavlockV/CoC7-FoundryVTT/blob/3fbae14bb8212bad4c1979a7888a9def1fd2ca74/module/actors/actor.js#L466-L468
suggestion ( Dex > Siz && Str >Siz):
```
468 : if( this.data.data.characteristics.dex.value > this.data.data.characteristics.siz.value && this.data.data.characteristics.str.value > this.data.data.characteristics.siz.value) MOV = 9; 
```

Let me ask another question.
Do the following mean "Major Wound" (in the rulebook p.120)?
https://github.com/HavlockV/CoC7-FoundryVTT/blob/3fbae14bb8212bad4c1979a7888a9def1fd2ca74/templates/actors/character-sheet.html#L114

And may I send you a pull request of the Japanese translation data (ja.json)?

I look forward to your reply.

**Comment by HavlockV** [comment]
Hi AdmiralNyar.
First of all thank you for the feedback, much appreciated.
Regarding the MOV calculation, you're perfectly right. I corrected the formula and it will be included in the next push.
criticalWounds is indeed a placeholder for the Major Wounds status. It's still not implemented and this stage just a switch. It will be hopefully soon operational as I'm almost done with the combat automation.
Finally you're more than welcome to send a pull request for a Japanese translation.

Thanks again for the feedback. I'll close the bug report once everything is corrected/pushed.

**Comment by HavlockV** [comment]
Wrong button.
Just reopening that issuer

**Comment by AdmiralNyar** [comment]
Thank you for checking.

I sent a pull request of Japanese localization files.

And, I'll send more Japanese translations when the localization is more advanced.

**Comment by HavlockV** [comment]
Japanese translation added in 0.2.3
Movement calculation corrected in 0.2.3
どうもありがとう ( i'm trying ;) )
